kod/object/item/passitem/spelitem.kod

```diff
@@ -76,6 +75,8 @@ properties:
 
    vrName = Spellitem_name_rsc
 
+   viObject_flags = APPLY_YES | GETTABLE_YES
```

Comment:
This line should be in wand.kod instead of spelitem.kod, otherwise it will cause potions and scrolls to have the targeting reticule also.